### ISSUE TITLE
Change PGP Release Key for Releases of `oscal-cli`

### ISSUE THREAD

**Issue by aj-stein-nist** [usnistgov/oscal-cli#99]
# User Story:

As an `oscal-cli` user, in order to ensure I have a signed release from a trusted identity from the NIST OSCAL Team, I would like an updated key belonging to a current developer or a release signing alias that belongs to appropriate members of the development team.

## Goals:

- [x] Replace Dave Waltermire's PGP key with another for signing releases in Maven as he has rolled off the project

## Dependencies:

{Describe any previous issues or related work that must be completed to start or complete this issue.}

## Acceptance Criteria

- [x] All website and readme documentation affected by the changes in this issue have been updated.
- [x] A Pull Request (PR) is submitted that fully addresses the goals of this User Story. This issue is referenced in the PR.
- [x] The CI-CD build process runs without any reported errors on the PR. This can be confirmed by reviewing that all checks have passed in the PR.

**Comment by aj-stein-nist** [comment]
I have updated documentation in #135 per recent community feedback but will need to update the README again when I shift to the new key for the next release.

**Comment by aj-stein-nist** [comment]
I updated with our new [GnuPG/PGP release key](https://pgp.mit.edu/pks/lookup?search=oscal%40nist.gov&op=index) and test with liboscal-java and oscal-cli. I might need to double check the oscal-cli but I am not sure I got the most recent artifact. Will test again before 0.3.4 but this looks like it will work off of SNAPSHOT files with the former, will test the latter, and write up key management procedures.
